yb-voyager/cmd/importSchema.go

```diff
@@ -118,6 +118,11 @@ func importSchema() {
 		installOrafceIfRequired(conn)
 	}
 
+	if !isYBDatabaseIsColocated(conn) && !utils.AskPrompt(fmt.Sprintf("\nWarning: Target DB '%s' is a non-colocated database, colocated tables can't be created in a non-colocated database.\n", tconf.DBName),
```

Comment:
This warning will not be correct if the colocated table list (as per assessment) is non-empty. Maybe we should add that as an additional condition?

Reply:
+ one nit: what Neeraj said in the meeting: if the --skip-recommendations was used, this would be redundant. 

Maybe we can handle this later (since it's just a warning, so it's harmless, user can always just continue)

Reply:
yeah, this warning is telling users to use colocated db only if tables are colocated...